examples/notebooks/Getting Started/Tutorial 3 - Basic Plotting.ipynb

```diff
@@ -733,6 +742,37 @@
     "sim.plot(quick_plot_vars=quick_plot_vars)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also plot multiple variables on the same plot by nesting lists"
```

Comment:
in this example the current densities don't change with time. do you think it would be better to plot a different quantity or pick a different model where they do change (e.g. DFN)? users might think the plot is broken if they aren't familiar with the model

Reply:
Good point! I changed the model to DFN now so the plot changes with time.